vendor/engines/badges_engine/spec/spec_helper.rb

```diff
@@ -0,0 +1,30 @@
+require 'simplecov'
+ENV["RAILS_ENV"] ||= 'test'
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.